### PR TITLE
B-1902 && B-1910 && B-1907 && B-1908

### PR DESCRIPTION
**Pull request by alexbeno**

### CODE REVIEW

File: app/home/components/page-grid/page-grid.tsx [hunk 1]
```diff
@@ -10,15 +10,18 @@ import styles from "app/home/styles/styles.module.css";
 import { viewportConfig } from "src/config";
 import { cn } from "src/utils/cn";
 
+import { useClientOnly } from "components/layout/client-only/client-only";
+
 import { TPageGrid } from "./page-grid.types";
 
 export function PageGrid({ children }: TPageGrid.Props) {
   const isLg = useMediaQuery(`(max-width: ${viewportConfig.breakpoints.lg}px)`);
```
Comment (haydencleary) [inline app/home/components/page-grid/page-grid.tsx:18]: If you use `useClientMediaQuery` this shouldn't be an issue

File: app/home/features/activity/activity.hooks.tsx [hunk 2]
```diff
@@ -78,9 +78,13 @@ export function useActivity() {
     }
   };
 
-  useRequestAnimationFrame(() => {
-    addActivityItem();
-  }, 5000);
+  useRequestAnimationFrame(
+    () => {
+      addActivityItem();
+    },
```
Comment (haydencleary) [inline app/home/features/activity/activity.hooks.tsx:84]:
```suggestion
      addActivityItem,
```

File: components/layout/container/container.tsx [hunk 3]
```diff
@@ -1,5 +1,5 @@
 import { PropsWithChildren } from "react";
 
 export function Container({ children }: PropsWithChildren) {
-  return <div className="mx-auto w-full max-w-7xl px-6">{children}</div>;
+  return <div className="mx-auto w-full max-w-7xl px-2 sm:px-6">{children}</div>;
```
Comment (haydencleary) [inline components/layout/container/container.tsx:4]: I think it's better to align the container padding with the header padding on mobile, no ?

**Review by haydencleary — CHANGES_REQUESTED** [review]

File: components/layout/container/container.tsx (hunk 3 above)
Comment (alexbeno) [reply components/layout/container/container.tsx:4]: yes but the 24px it quit large for a mobile container

**Review by haydencleary — APPROVED** [review]

**Pull request merged**